Cargo.toml

```diff
@@ -1,28 +1,36 @@
 [workspace]
 resolver = "2"
-members = ["demo"]
+members = ["examples/with_winit", "examples/run_wasm", "examples/scenes"]
 
 [workspace.package]
 edition = "2021"
-version = "0.0.1"
-license = "MIT OR Apache-2.0"
+version = "0.1.0"
+license = "Apache-2.0 OR MIT"
 repository = "https://github.com/linebender/velato"
 
 [package]
 name = "velato"
-description = "Lottie renderer built on vello."
+description = "A Lottie integration for vello."
 categories = ["rendering", "graphics"]
-keywords = ["2d", "vector-graphics", "animation", "lottie"]
-
+keywords = ["2d", "vector-graphics", "vello", "animation", "lottie"]
 version.workspace = true
 license.workspace = true
 edition.workspace = true
 repository.workspace = true
 
 [workspace.dependencies]
-vello = { git = "https://github.com/linebender/vello", rev = "b0303ccf98df15a8b196272720d364a56f7304ba" }
-
+# Update the README badges to match wgpu and vello version!
+vello = "0.1"
 
 [dependencies]
 vello = { workspace = true }
-bodymovin = { git = "https://github.com/vectorgameexperts/bodymovin-rs" }
+keyframe = "1"
+once_cell = "1"
+
+# For the parser
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+serde_repr = "0.1"
+
+[target.'cfg(target_arch = "wasm32")'.dev-dependencies]
+wasm-bindgen-test = "0.3"
```

Comment:
Let's specify the full version because otherwise `cargo upgrade` will ignore patch updates. Also, some of these are really old, e.g. serde 1.0 was released in 2017. Someone might try to actually use it via `-Z minimal-versions` and we don't test it.

I'll note that I haven't actually tested this combination locally. Hopefully no deep dependency has a pinned sub dependency which prevents using the latest versions. Our CI will catch that in this PR if that's the case though.

```suggestion
keyframe = "1.1.1"
once_cell = "1.19.0"

# For the parser
serde = { version = "1.0.197", features = ["derive"] }
serde_json = "1.0.114"
serde_repr = "0.1.18"

[target.'cfg(target_arch = "wasm32")'.dev-dependencies]
wasm-bindgen-test = "0.3.42"
```